### ISSUE TITLE
fix 

### ISSUE THREAD

**Issue by lanzzbr** [B3akers/LeagueSkinChanger#71]
Open source, locate "autoupdater.cpp" search for "8A 87 ? ? ? ? 88 4C 24 **0B**" inside "autoupdater.cpp" change to "8A 87 ? ? ? ? 88 4C 24 **?**"... Save, build in release x86, done... Thanks to @R3nzTheCodeGOD

**Comment by superchedder** [comment]
so we need to compile it ?

**Comment by lanzzbr** [comment]
> so we need to compile it ?

yes, if you want to use it now, or wait, for B3akers to update it.

**Comment by enescglyn** [comment]
thx <3
